app/uk/gov/hmrc/customs/notification/services/UnblockPollerService.scala

```diff
@@ -67,7 +67,6 @@ class UnblockPollerService @Inject()(config: CustomsNotificationConfig,
 
   private def pushOrPull(workItem: WorkItem[NotificationWorkItem]): Future[Boolean] = {
 
-    implicit val loggingContext: NotificationWorkItem = workItem.item
```

Comment:
Just wondering if might affect anything as implicit are very tricky to spot in terms of usage

Reply:
Not impacting tests wise, but we know it can be a potential problem so when anything goes wrong we will be in a position to undo it.